packages/core/src/components/Chip.story.tsx

```diff
@@ -12,10 +14,51 @@ storiesOf('Core/Chip', module)
     inspectComponents: [Chip],
   })
   .add('Simple chip.', () => <Chip>Chip</Chip>)
+  .add('With a icons before or after.', () => (
+    <>
+      <Spacing right={1} inline>
+        <Chip icon={<IconCloseAlt size="2em" />}>Chip</Chip>
+      </Spacing>
+
+      <Spacing right={1} inline>
+        <Chip beforeIcon={<IconCalendar size="1.2em" />}>Calendar</Chip>
+      </Spacing>
+
+      <Spacing right={1} inline>
+        <Chip icon={<IconCloseAlt size="2em" />} beforeIcon={<IconUser size="1.2em" />}>
+          Calendar
+        </Chip>
+      </Spacing>
+
+      <Spacing right={0} inline>
+        <Chip icon={<IconCloseAlt size="2em" />} beforeIcon={<IconCalendar size="1.2em" />}>
```

Comment:
should these examples use `afterIcon` to encourage that usage instead of `icon`?

Reply:
aaah or can both be supported? looks like both

Reply:
Yeah, temporarily trying to preserve backwards compatibility until the next major release :(
You're right though, `afterIcon` is preferred, switching everything over, thanks!